Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/InternetExplorer/InternetExplorerSpecificAccessibilityViolationsExtensions.cs

```diff
@@ -0,0 +1,49 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using JetBrains.Annotations;
+using Remotion.Utilities;
+using Remotion.Web.Development.WebTesting.Accessibility;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure.InternetExplorer
+{
+  public static class InternetExplorerSpecificAccessibilityViolationsExtensions
+  {
+    /// <summary>
+    /// Removes elements where known workarounds cause violations.
+    /// </summary>
+    public static IEnumerable<AccessibilityRuleResult> IgnoreKnownWorkarounds (
+        [NotNull] this IEnumerable<AccessibilityRuleResult> violations,
+        [NotNull] IBrowserConfiguration browserConfiguration)
+    {
+      ArgumentUtility.CheckNotNullOrItemsNull ("violations", violations);
+      ArgumentUtility.CheckNotNull ("browserConfiguration", browserConfiguration);
+
+      if (browserConfiguration.IsInternetExplorer())

```

Comment:
here, we have an extension method IsInternetExplorer => those should be on the same type and then it's easy to discover all usages.